content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+6. The default settings for _filmic_ are adjusted by the software according to the exposure bias stored in the photo metadata. Thus, for digital photographers who are accustomed to exposing to the right of the histogram, forcing a manual exposure compensation on the camera body to protect the highlights from clipping (which is the only good way to expose in digital photography, with all due respect to purists who don't understand the technological differences between photoelectric semiconductor and photo-sensitized silver halide ions) will have their pipeline, relative to the scene, set automatically for them (as a first approximation, because filmmakers still can't guess what the exact mid-grey value is for a given scene).
+
+_Filmic v3_ is still available via the last tab of _filmic rgb_. As well as it is possible to update an image from _filmic v3_ to v4 with the same tab. Here a preview of this new _filmic rgb_ and the selection of the version wanted via science color menu:
+
+  <div style="text-align: center">
+![filmic rgb v4]({filename}en/filmicv4.png)
+  </div>
+
+Note: the reconstruction of highlights is done by means of wavelet decomposition by iteration, which induces a certain performance penalty. It is possible to disable the reconstruction completely by setting a very high highlight threshold, as the program bypasses the reconstruction if no pixels are detected beyond this threshold. Nevertheless, given the difficulty of convincingly reconstructing highlights, this is an acceptable compromise.
+
+It is also possible to deactivate, in some cases, the default _higlight reconstruction_ module if _filmic_ reconstruct itself the highlights, to preserve more details. Then think of controlling the desaturation of the highlights in order to absorb the magenta highlights that you could create by deactivating this module.
+
+# Others improvements
+
+This 3.2, like the 3.0, is so rich in new features and improvements. This article doesn't fit to be exhaustive but to present the main changes and allow you to understand them. You will find below, in a jumble, a few other improvements to know.
```

Comment:
"... is **extremely rich** in new features..."

"This article **isn't intended to be exhaustive**..."